Prepare for scidata submission (#196)
myyoda/principles-paper: scidata-cover-letter.md

# Cover letter — Scientific Data submission

<!-- Uploaded as the covering-letter file on SNAPP (a technical requirement of the system).
Per the journal's guidance, cover-letter content is not used for editorial decisions,
so this letter carries logistics only: scope statement, corresponding author,
reviewer suggestions, and disclosures. Data-access details belong in the manuscript,
never here (reviewers do not see this file). -->

Dear Editors,

We submit "STAMPED principles for reproducible research objects" for consideration as an Article in Scientific Data.

> The manuscript formalizes seven principles (Self-containment, Tracking, Actionability, Modularity, Portability, Ephemerality, and Distributability: STAMPED) for structuring and managing research objects so that they can be verified, re-executed, and extended.

Reviewer: I would add some preamble like "Based on our years of experience ..." or alike

> It addresses the mechanics of data organization, sharing, and reuse, and we believe it falls squarely within the Article scope.

Reviewer: I think it might be worth stating that it is not conflicting or specializing FAIR but rather is complimentary to FAIR principles, the seminal paper on which was published in SciData.

We are aware the submission guidelines advise against self-constructed acronyms in titles.
We have retained STAMPED in the title because introducing the acronym is the purpose of the paper, following the precedent of the FAIR Guiding Principles (https://doi.org/10.1038/sdata.2016.18) and the TRUST Principles (https://doi.org/10.1038/s41597-020-0486-7), both published in Scientific Data with the acronym they introduce in the title.
We are happy to discuss alternatives if the editors prefer.

Corresponding author: Austin Macdonald, Dartmouth College (austin@dartmouth.edu, ORCID: 0000-0002-8124-807X).

Suggested reviewers:
<!-- PENDING (Yarik): 3–5 entries — name, affiliation, email, one-line rationale.
Journal auto-checks public conflicts (recent co-authorship, shared institution); suggestions are optional and do not expedite review, but are welcome. -->

> - TBD

Reviewer: @yarikoptic will suggest some with account to his COI collabs.tsv

It addresses the mechanics of data organization, sharing, and reuse, and we believe it falls squarely within the Article scope.

STAMPED complements the FAIR Guiding Principles (https://doi.org/10.1038/sdata.2016.18), whose seminal paper was published in Scientific Data.
FAIR addresses discovery and reuse across repositories, while STAMPED describes principles for how the research object itself is built and run.

We are aware the submission guidelines advise against self-constructed acronyms in titles.
We have retained STAMPED in the title because introducing the acronym is the purpose of the paper, following the precedent of the FAIR principles cited above and the TRUST Principles (https://doi.org/10.1038/s41597-020-0486-7), both published in Scientific Data with the acronym they introduce in the title.
We are happy to discuss alternatives if the editors prefer.

Corresponding author: Austin Macdonald, Dartmouth College (austin@dartmouth.edu, ORCID: 0000-0002-8124-807X).

Suggested reviewers:
<!-- PENDING (Yarik): 3–5 entries — name, affiliation, email, one-line rationale, screened against his collaborator list for conflicts.

> Journal auto-checks public conflicts (recent co-authorship, shared institution); suggestions are optional and do not expedite review, but are welcome. -->

Reviewer: wow, my query to claude as so good that the first candidate as Carole Goble as she was not explicitly listed in my list of COIs :-/    So -- that is good as she would have been a great reviewer. But also it points that all candidates below are high profile so - expect fire!

And now I have a connondrum that I better not publish their emails here publicly since some of them did go lengths to obfuscate them, thus listing without emails and will paste info with emails to matrix

```suggestion
<!-- DONE: name, affiliation, email, one-line rationale, screened against his collaborator list for conflicts.
Journal auto-checks public conflicts (recent co-authorship, shared institution); suggestions are optional and do not expedite review, but are welcome. -->
 
1. **Stian Soiland-Reyes** — Univ. of Manchester (also Univ. of Amsterdam) — `EMAILTODO`* — Co-editor of the RO-Crate specification cited in the paper; expert on research-object packaging and workflow provenance.
2. **Neil Chue Hong** — Software Sustainability Institute / EPCC, Univ. of Edinburgh — `EMAILTODO` — FAIR4RS Principles co-author; RSE-community and research-software-sustainability lead.
3. **Sarah Cohen-Boulakia** — LISN, CNRS / Univ. Paris-Saclay — `EMAILTODO` — 2024 CNRS Silver Medal; IUF senior member; coordinates ShareFAIR; workflow reuse and reproducibility in life sciences.
4. **Konrad Hinsen** — Centre de Biophysique Moléculaire, CNRS, Orléans — `EMAILTODO` — co-EiC of *ReScience C*; two decades of computational-reproducibility formalization (ActivePapers, Guix-in-science).
5. **Ewa Deelman** — USC Information Sciences Institute; USC Computer Science — `EMAILTODO` — 2025 IEEE Sidney Fernbach Award for scientific workflows (Pegasus WMS); authority on Actionability/Portability/Ephemerality.
6. **Kyle Chard** — Univ. of Chicago; Argonne National Laboratory (Globus Labs, Parsl) — `EMAILTODO` — Co-author of the 2026 FGCS paper "A terminology for scientific workflow systems", a directly analogous vocabulary-standardization effort.
```

- TBD

Non-preferred reviewers: none.

<!-- PENDING: disclose here any live collaborations related to the topic that are not
discoverable from public records (the one disclosure the journal asks for in letters);

> delete this block if none. -->

Reviewer: list Carole Goble here .  She is not a collaborator per se but I think there is COI due to her part on EAB.

2. Neil Chue Hong — Software Sustainability Institute / EPCC, Univ. of Edinburgh — FAIR4RS Principles co-author; RSE-community and research-software-sustainability lead.
3. Sarah Cohen-Boulakia — LISN, CNRS / Univ. Paris-Saclay — 2024 CNRS Silver Medal; IUF senior member; coordinates ShareFAIR; workflow reuse and reproducibility in life sciences.
4. Konrad Hinsen — Centre de Biophysique Moléculaire, CNRS, Orléans — co-EiC of *ReScience C*; two decades of computational-reproducibility formalization (ActivePapers, Guix-in-science).
5. Ewa Deelman — USC Information Sciences Institute; USC Computer Science — 2025 IEEE Sidney Fernbach Award for scientific workflows (Pegasus WMS); authority on Actionability/Portability/Ephemerality.
6. Kyle Chard — Univ. of Chicago; Argonne National Laboratory (Globus Labs, Parsl) — Co-author of the 2026 FGCS paper "A terminology for scientific workflow systems", a community taxonomy of workflow management systems.

Due to a possible conflict of interest, we suggest the following reviewer not be invited:

> - Carole Goble — Univ. of Manchester — member of the External Advisory Board of ReproNim, which funds this work (listed in the manuscript's Funding section).

Reviewer: ```suggestion
- Carole Goble — Univ. of Manchester — member of the External Advisory Board of NIH P41 ReproNim Center project, which is among funders of this work (listed in the manuscript's Funding section).
```

to give more value and also state that it is not the only funding source

On behalf of the authors, thank you for your consideration.

> Austin Macdonald (corresponding author)\

Reviewer: ```suggestion
Austin Macdonald (corresponding author)
```

Author: this was deliberate to force a newline. Do you intend my name and all afflications on 1 line?
